apps/src/lib/tools/jsdebugger/JsDebugger.story.jsx

```diff
@@ -23,13 +23,19 @@ export default storybook => {
 
   storyTable.push({
     name: 'empty',
-    story: () => (
-      <Provider store={createApplabStore()}>
-        <JsDebugger
-          style={storybookStyle}
-        />
-      </Provider>
-    )
+    story: () => {
+      const withDebugConsoleStore = createApplabStore();
+      withDebugConsoleStore.dispatch(setPageConstants({
+        appType: "gamelab"
+      }));
+      return (
+        <Provider store={withDebugConsoleStore}>
```

Comment:
This looks good.  Not to change now, but wanted to call out that this whole story file is probably a good candidate for the `withReduxStore` storybook extension we wrote recently. ([definition](https://github.com/code-dot-org/code-dot-org/blob/414c5e04e718c067b54c360c21f4fdf7f8952e9f/apps/test/util/withReduxStore.js), [example use](https://github.com/code-dot-org/code-dot-org/blob/94e5666cffbc8261221985a4bb5fa1a67a097fd9/apps/src/code-studio/components/ShareAllowedDialog.story.jsx#L24))